src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -274,7 +274,7 @@
         {% endif %}
         {{ extraVars.label }}
 
-        {% if form.vars.label_attr and form.vars.label_attr['popover'] %}
+        {% if form.vars.label_attr is defined and form.vars.label_attr and form.vars.label_attr['popover'] %}
           {{ include('@Common/HelpBox/helpbox.html.twig', {'content': form.vars.label_attr['popover']}) }}
```

Comment:
Not sure if we still need and form.vars.label_attr part, I guess it's helpful to check it's not false.

Reply:
Always good to check 🎉